docs/mmteb/points/637.jsonl

```diff
@@ -0,0 +1 @@
+{"GitHub": "sted97", "New dataset": 6}
```

Comment:
please add review points and we'll be able to merge it. Thanks for your contribution 🙂

Reply:
Done, thanks!